src/algorithms/stats/histogram.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "histogram.h"
+
+using namespace essentia;
+using namespace standard;
+
+const char* Histogram::name = "Histogram";
+const char* Histogram::category = "Statistics";
+const char* Histogram::description = DOC("This algorithm computes a histogram given the following preconfigured parameters: a normalization setting, a minimum value, a maximum value, and the number of equal sized bins for the histogram. ");
```

Comment:
No need to describe the parameters in the DOC string, as they will be picked up for documentation from the ```declareParameter()``` methods.